Index: mplayer-dmo-1.0~rc4+svn20120511/etc/example.conf
===================================================================
--- mplayer-dmo-1.0~rc4+svn20120511.orig/etc/example.conf	2012-05-11 08:47:00.678057116 +0200
+++ mplayer-dmo-1.0~rc4+svn20120511/etc/example.conf	2012-05-11 08:47:05.290226641 +0200
@@ -74,7 +74,7 @@
 ##################
 
 # Specify default audio driver (see -ao help for a list).
-#ao=oss
+ao=alsa,
 
 # Use SDL audio driver with the esd subdriver by default.
 #ao = sdl:esd
@@ -145,6 +145,11 @@
 # DVD: Play English audio tracks if available.
 #alang = en
 
+# get a default OSD font from fontconfig
+
+fontconfig = yes
+font = "Sans"
+subfont-text-scale = 3
 
 ############
 # Profiles #
@@ -167,5 +172,6 @@
 # OSD progress bar vertical alignment
 #progbar-align=50
 
+[default]
 # You can also include other configuration files.
-#include = /path/to/the/file/you/want/to/include
+include = /etc/mplayer/mplayer.conf.local
